scripts/us_epa/parent_company/README.md

```diff
@@ -52,22 +52,62 @@ percentage and if no percentage is provided it is assumed to be 100.
     warnings for 'Existence_MissingReference_Property' are for the parent
     company names not yet existing in Data Commons.
 
+    The counters produced for the company table import followed by the
+    ownership import:
+
     ```
     "levelSummary": {
       "LEVEL_INFO": {
         "counters": {
           "NumRowSuccesses": "102603",
           "NumPVSuccesses": "820824",
-          "Existence_NumChecks": "785041",
+          "Existence_NumChecks": "765010",
           "NumNodeSuccesses": "102603",
           "Existence_NumDcCalls": "7"
         }
       },
       "LEVEL_WARNING": {
         "counters": {
           "Existence_MissingTriple_domainIncludes": "1",
-          "Existence_MissingReference_locatedIn": "685"
+          "Existence_MissingReference_locatedIn": "513"
         }
       }
     },
     ```
+
+    For the Stat Var Observations, there are several warnings for major jumps
```

Comment:
Should we update the Resolution section above with the new fancier dup detection? (could link to a PDF export of the doc you have?)